portfolio/src/main/webapp/style.css

```diff
@@ -1,17 +1,20 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
-}
-
-#greeting-container {
-  margin-top: 50px;
+  margin-top: 35px;
+  margin-bottom: 65px;
+  padding-top: 35px;
+  padding-bottom: 50px;
+  padding-left: 50px;
+  padding-right: 50px;
```

Comment:
[Style](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties): Use shorthand properties where possible.

Here and elsewhere.

This page has a list of shorthand properties:
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties